templates/staff/application/researcher_edit.html

```diff
@@ -0,0 +1,107 @@
+{% extends "staff/base-tw.html" %}
+
+{% block metatitle %}Edit {{ researcher.name }}: Staff Area | OpenSAFELY Jobs{% endblock metatitle %}
+
+{% block breadcrumbs %}
+{% #breadcrumbs %}
+  {% url "staff:index" as staff_url %}
+  {% url "staff:application-list" as staff_application_list_url %}
+  {% breadcrumb title="Staff area" url=staff_url %}
+  {% breadcrumb title="Applications" url=staff_application_list_url %}
+  {% breadcrumb title="Application: "|add:researcher.application.pk_hash url=researcher.application.get_staff_url|add:"#researcher-details" %}
+  {% breadcrumb title="Edit researcher: "|add:researcher.name active=True %}
+{% /breadcrumbs %}
+{% endblock breadcrumbs %}
+
+{% block hero %}
+{% staff_hero title="Edit Researcher: "|add:researcher.name %}
+{% endblock hero %}
+
+{% block content %}
+<form method="POST" class="flex flex-col gap-8">
+  {% csrf_token %}
+
+  {% if form.non_field_errors %}
+    {% for error in form.non_field_errors %}
+      {% #alert variant="danger" class="mb-6" %}
+        {{ error }}
+      {% /alert %}
+    {% endfor %}
+  {% endif %}
+
+  {% #card title="Researcher contact information" class="max-w-3xl" container=True %}
+    {% #form_fieldset class="grid gap-y-6 !min-w-0 w-full max-w-prose" %}
+      {% form_legend text="Researcher contact information" class="sr-only" %}
+
+      {% form_input field=form.name label="Researcher name" name="name" %}
+      {% form_input field=form.job_title label="Researcher job title" name="job_title" %}
+      {% form_input field=form.email label="Researcher work email address" name="email" type="email" %}
+      {% form_input field=form.telephone label="Mobile phone number" name="telephone" %}
+      {% form_input field=form.github_username label="GitHub username" name="github_username" %}
+      {% form_input field=form.daa label="DAA" name="daa" %}
+    {% /form_fieldset %}
+  {% /card %}
+
+  {% #card title="Secure results server" class="max-w-3xl" container=True %}
```

Comment:
If there's a validation error somewhere on the page when I press submit, then the page refreshes and loses this value.

I can then submit without answering this question - I'm not sure if that's intentional or not.

Reply:
Huh, it seems like when the if statement is `False == False` the `isChecked` variable is not being set.